bounties/level-1/basic-account-viewer.md

```diff
@@ -0,0 +1,51 @@
+# Basic account viewer
+
+[🔵 Start hunting](https://github.com/tyvdh/stellar-quest-bounties/issues/new?assignees=&labels=&template=begin-the-hunt.md&title=%F0%9F%94%B5+%60basic-account-viewer%60) this bounty
+
+## Award
+| Hunter | Reviewer | Type
+| :- | :- | :-
+| 200 XLM | 1 Credits | Continuos
+
+## Context
+https://github.com/tyvdh/stellar-quest-bounties/issues?q=basic-account-viewer.md
+
+## Description
+
+### What is this task?
+
+When building websites ontop of Stellar, there needs to be a way to access a user's Stellar account. This is where web-wallets like [Albedo](https://albedo.link/), [Rabet](https://rabet.io/) and [Freighter](https://www.freighter.app/) come in. They allow websites to easily and securely acces your Stellar account.
+
+Your task is to build a simple website which allows you to sign in and log out using a web-wallet like Albedo, Freighter or Rabet.
+
+The website should show some basic information about the account currently logged in:
+ * The asset-codes and balances of the account's assets should be shown.
+ * The date when the account was created should be shown.
+ * The account who created this account should be shown.
+
+For this bounty, the UI is not important. However CSS and HTML should be used to create a simple layout. Additionally, the solution should be licensed as open source.
+ 
+### What are the requirements for the bounty hunter?
+* The hunter should be able to build simple websites with Javascript.
+
+### What are the deliverables?
+#### Description
+  - The hunter should provide a link to the repository containing the solution.
+  - The hunter should publish the website and provide a link to the published website.
+
+#### Review Criteria
+  - UI is not important, but there should atleast be a basic layout.
+  - The code quality is not important.
```

Comment:
The code quality is also not primarily important but should function properly and be reasonably reviewable.